src/libraries/UniswapV3Lib.sol

```diff
@@ -11,20 +11,30 @@ import { IRateLimits }                                                  from "..
 import { ISwapRouter, IUniswapV3PoolLike, INonfungiblePositionManager } from "../interfaces/UniswapV3Interfaces.sol";
 
 import { FullMath }   from "lib/dss-allocator/src/funnels/uniV3/FullMath.sol";
-import { UniV3Utils } from "lib/dss-allocator/test/funnels/UniV3Utils.sol";
 import { TickMath }   from "lib/dss-allocator/src/funnels/uniV3/TickMath.sol";
+import { LiquidityAmounts } from "lib/dss-allocator/src/funnels/uniV3/LiquidityAmounts.sol";
```

Comment:
Alignment

I mentioned this in another PR, but I don't think we should import and use `UniV3Utils` from `dss-allocator` since it is coming from the `test` directory.  We can use the code, but it needs to be copied into our libraries to ensure it is included properly in an audit scope